dotenv/src/find.rs

```diff
@@ -51,7 +51,7 @@ pub fn find(directory: &Path, filename: &Path) -> Result<PathBuf> {
     } else {
         Err(Error::Io(io::Error::new(
             io::ErrorKind::NotFound,
-            "path not found",
+            ".env file not found inside parent",
```

Comment:
The filename could be *custom.env*.

May I suggest:
`dotenv file not found in parent directory`

Reply:
Great point, I think that'd work well!